src/lib/cloud/digitalocean/provisioner.ts

```diff
@@ -33,13 +31,89 @@ const REDIRECT_URIS: [string] = [
   //  'http://localhost:10101'
 ];
 
+const STORAGE_KEY_OAUTH = 'DigitalOcean-OAuth';
+
 interface KeyPair {
   private: string;
   public: string;
 }
 
 class Provisioner {
-  constructor(private dispatch_: Function, private state_: any = {}) {}
+  constructor(
+    private sshKeyPair_ :any = {},
```

Comment:
I fully support removing the `state_` field. A couple of things though:
 * we should keep `dispatch_` which is an argument passed implicitly to all freedomjs module constructors
 * let's define these fields normally, i.e. outside the constructor